pkg/api/entries.go

```diff
@@ -438,3 +421,36 @@ func SearchLogQueryHandler(params entries.SearchLogQueryParams) middleware.Respo
 
 	return entries.NewSearchLogQueryOK().WithPayload(resultPayload)
 }
+
+// Attempt to retrieve a UUID from a backend tree
+func RetrieveUUID(params entries.GetLogEntryByUUIDParams, uuid string, tid int64) (middleware.Responder, error) {
```

Comment:
I think it would still make more sense to return `(models.LogEntry,, error)` here and create the response in `GetLogEntryByIndexHandler`

Reply:
Looks like creating a `models.LogEntry` will require turning a [`trillian.GetEntryAndProofResponse`](https://pkg.go.dev/github.com/google/trillian#GetEntryAndProofResponse) into a `models.LogEntryAnon`, is that right? Is there a straightforward way to do that? I'm looking at [`logEntryFromLeaf`](https://github.com/sigstore/rekor/blob/6cef9e450975125b9e372a4d7488c63bf4bfd8ad/pkg/api/entries.go#L66) as an example but not really seeing how the fields match up.

Reply:
Make a `LogEntryAnon` and unmarshal into it?

Reply:
I think `logEntryFromLeaf` returns `models.LogEntry`, so after line [443](https://github.com/sigstore/rekor/pull/750/files#diff-d20fa2d682d00a31fa8535c7b3607012adab9f6fa9151768090a6dfab1a5b9b0R443) you could just `return logEntry, nil`?

Reply:
Nice! You're so right. Thanks!